scripts/import_gff3.py

```diff
@@ -0,0 +1,347 @@
+# -*- coding: utf-8 -*-
+
+import pandas as pd
+from collections import defaultdict
+
+"""
+Function that loads a gff file into a pandas dataframe
+"""
+def loadfile(filename,verbose):
+        
+    try:
+        if verbose==True:
+            print 'Loading', filename
+        # obtaning sample names and number from 3rd line in header
+        with open(filename) as f:
+            f.readline()
+            f.readline()
+            sample_names = f.readline().split()[2].split(',')
```

Comment:
samples can be in different lines. Better to find the samples by COLDATA. You can use https://github.com/miRTop/mirtop/blob/dev/mirtop/gff/header.py#L26 to get a list of the samples name.

Reply:
Should I import the function read_samples(fn) from header.py or simply use the code?